redhat-cop/containers-quickstarts
# Use UBI for jenkins-slave-ruby and add to testing

#### What is this PR About?
Use UBI for jenkins-slave-ruby and add test for image.

#### How do we test this?
```
./_test/setup.sh applier jenkins-slaves pabrahamsson/containers-quickstarts ruby_agent_ubi
./_test/setup.sh test jenkins-slaves
Ensure all Builds are executed...
build.build.openshift.io/jenkins-slave-ansible-pipeline-1 started
build.build.openshift.io/jenkins-slave-golang-pipeline-1 started
build.build.openshift.io/jenkins-slave-ruby-pipeline-1 started
Waiting for all builds to start...
Waiting for all builds to complete...
Check to see how many builds Failed
Tests Completed Successfully!
```

cc: @redhat-cop/day-in-the-life
